core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3322,49 +3369,55 @@ private SyncResult syncInodeMetadata(RpcContext rpcContext, LockedInodePath inod
         }
       }
       if (syncPlan.toLoadMetadata()) {
-        loadMetadata = true;
+        AlluxioURI mountUri = new AlluxioURI(mMountTable.getMountPoint(inodePath.getUri()));
+        pathsToLoad.add(mountUri.getPath());
       }
       if (syncPlan.toSyncChildren() && inode instanceof InodeDirectory
           && syncDescendantType != DescendantType.NONE) {
         UfsStatus[] listStatus = ufs.listStatus(ufsUri.toString());
```

Comment:
Can you move this down a few lines to right before it is checked/used?

Reply:
done